spec/services/imports/lettings_logs_import_service_spec.rb

```diff
@@ -671,5 +671,47 @@ def open_file(directory, filename)
         expect(lettings_log.hhmemb).to eq(1)
       end
     end
+
+    context "when there are no outstanding charges but outstanding amount is given" do
+      let(:lettings_log_id) { "0b4a68df-30cc-474a-93c0-a56ce8fdad3b" }
+      let(:lettings_log_file) { open_file(fixture_directory, lettings_log_id) }
+      let(:lettings_log_xml) { Nokogiri::XML(lettings_log_file) }
+
+      before do
+        FormHandler.instance.use_fake_forms!
+        Singleton.__init__(FormHandler)
```

Comment:
minor thing but i'm pretty sure you don't need to reset the instance every time and calling `use_fake_forms!` should be enough to mutate the instance. but you will need to call `use_real_forms!` afterwards as you are already doing so

Reply:
Yeah, I was a bit confused, it didn't work without me resetting it. So `FormHandler.instance.current_lettings_form` worked as expected, but the `FormHandler.instance.get_form("current_lettings")` didn't. Even if I put the content of the `FormHandler.instance.current_lettings_form` in `FormHandler.instance.get_form(name)`, it still didn't return the updated form